package.json

```diff
@@ -57,6 +57,26 @@
         "gulp-uglify": "3.0.1"
     },
     "config": {
-        "posts_per_page": 25
+        "posts_per_page": 25,
+        "image_sizes": {
+            "xxs": {
```

Comment:
Is there a use-case for this `xxs` size or is it there more as an example?

Reply:
actively used for author images @1x and in floating-header.hbs for site icon

Reply:
`floating-header.hbs` is using `xs` at the moment, I asked because I couldn't see `xxs` being used anywhere - wondered if it was left in from earlier dev or missed out somewhere else 🙂